static/app/gettingStartedDocs/electron/electron.tsx

```diff
@@ -0,0 +1,98 @@
+import ExternalLink from 'sentry/components/links/externalLink';
+import {Layout, LayoutProps} from 'sentry/components/onboarding/gettingStartedDoc/layout';
+import {ModuleProps} from 'sentry/components/onboarding/gettingStartedDoc/sdkDocumentation';
+import {StepType} from 'sentry/components/onboarding/gettingStartedDoc/step';
+import {getUploadSourceMapsStep} from 'sentry/components/onboarding/gettingStartedDoc/utils';
+import {t, tct} from 'sentry/locale';
+
+export const steps = ({
+  sentryInitContent,
+}: {
+  sentryInitContent?: string;
+} = {}): LayoutProps['steps'] => [
+  {
+    type: StepType.INSTALL,
+    description: t('Add the Sentry Electron SDK package as a dependency:'),
+    configurations: [
+      {
+        language: 'bash',
+        code: `
+# Using yarn
+yarn add @sentry/electron
+
+# Using npm
+npm install --save @sentry/electron
+        `,
+      },
+    ],
+  },
+  {
+    type: StepType.CONFIGURE,
+    description: (
+      <p>
+        {tct(
+          `You need to call [codeInit:Sentry.init] in the [codeMain:main] process and in every [codeRenderer:renderer] process you spawn.
+           For more details about configuring the Electron SDK [docsLink:click here].`,
+          {
+            codeInit: <code />,
+            codeMain: <code />,
+            codeRenderer: <code />,
+            docsLink: (
+              <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/electron/" />
+            ),
+          }
+        )}
```

Comment:
```suggestion
        {tct(
          `You need to call [code:Sentry.init] in the [code:main] process and in every [code:renderer] process you spawn.
           For more details about configuring the Electron SDK [docsLink:click here].`,
          {
            code: <code />,
            docsLink: (
              <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/electron/" />
            ),
          }
        )}
```

Reply:
Hmm I tried this initially but as soon as use `code` in all three cases, the last value ('renderer') is applied to all of them. 

![image](https://github.com/getsentry/sentry/assets/8420481/d93f3bd6-a9fa-4993-9b31-012e02c1de93)

Any ideas how to fix this properly?

Reply:
oh no.... now that I remember, I ran into the same problem... we will have to investigate, so for now let's leave it as it is 🙂